conf/application.conf

```diff
@@ -234,3 +234,24 @@ hbase {
   initialize = false
   initialize = ${?HBASE_INITIALIZE_DB}
 }
+
+kamon {
+  environment {
+    service = "sbr-admin-data"
+  }
+
+  trace {
+    join-remote-parents-with-same-span-id = true
+    sampler = always
+    sampler = ${?SBR_TRACING_SAMPLER}
+  }
+
+  reporters = ["kamon.zipkin.ZipkinReporter"]
+
+  zipkin {
+    host = "localhost"
+    host = ${?SBR_TRACING_REPORTER_HOST}
+    port = 9411
+    port = ${?SBR_TRACING_REPORTER_PORT}
+  }
+}
```

Comment:
In order to consider this story "done done", we should make sure that the correct Zipkin configuration is added to the Gitlab manifest files.

(Note that the Gitlab project used is confusingly the one for the old name - i.e the one ending -api).